Documents/working_draft_wikipedia_page.md

```diff
@@ -0,0 +1,115 @@
+# SARIF
+The **Static Analysis Results Interchange Format** (**SARIF**; pronunciation_guide) is a universal representation for the output of [static analysis](https://en.wikipedia.org/wiki/Static_analysis) tools.  It is standardized by the [OASIS](https://en.wikipedia.org/wiki/OASIS_(organization)) [SARIF Technical Committee](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=sarif).
+
+### Contents
```

Comment:
we could change to `## Contents` and for each item in the topic start with ``### Topic`. If the topic has sub-topics, use `####`.

This would make the titles to be in the correct heading format.

Reply:
@eddynaka - The markdown implementation is not really important because it must all be translated to Wikipedia, where it will be changed into their format.

Reply:
I see!
thanks for explaining! :)